lib/const.rb

```diff
@@ -36,10 +36,13 @@
 end
 
 if ENV['CREW_CACHE_DIR'].to_s == ''
-  CREW_CACHE_DIR = HOME + '/.cache/crewcache/'
+  CREW_CACHE_DIR = HOME + '/.cache/crewcache'
 else
   CREW_CACHE_DIR = ENV['CREW_CACHE_DIR']
 end
+FileUtils.mkdir_p CREW_CACHE_DIR unless Dir.exist? CREW_CACHE_DIR
+
+CREW_IS_CACHING = ENV['CREW_CACHE_OPT']
```

Comment:
It appears to be a bit confusing to have an environment variable and crew constant with different names.  Can we just rename one or the other to match?

Reply:
Sure.